### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by chrisbra** [Slesa/launchy#10]
Hi,
is this project still active? Or where is the main application now developed?

**Comment by Slesa** [comment]
Hi,
right now I have too much trouble at work to continue work. I don't know if there are other projects, but I assume this is the latest approach to use Qt5. Do you need something special?

**Comment by chrisbra** [comment]
I was browsing the forum for launchy, looking for help and then I found out about your project but it also looked abandoned, so I wondered if it makes sense to ask about launchy at all. I now found Keypirinha, which seems to be an actively maintained alternative project for launchy.

**Comment by Slesa** [comment]
Yes, Keypirinha looks rather pretty. It was a reason to ask: why to continue launchy at all. One reason would be using Qt5.

**Comment by chrisbra** [comment]
To be honest, I don't particular care for the underlying windowing framework. I understand, that this is a rather important decision from a development perspective, especially if one wants to be cross-plattform compatible. But from a user perspective I don't care much (at least on Windows)

**Comment by Slesa** [comment]
Yes, yes - no doubt about that. If you're fine with Keypirinha, that is completely ok.
